lib/cfg-grammar.y

```diff
@@ -1189,6 +1191,8 @@ threaded_dest_driver_option
         {
           log_threaded_dest_driver_set_max_retries(last_driver, $3);
         }
+        | KW_BATCH_LINES '(' nonnegative_integer ')' { log_threaded_dest_driver_set_flush_lines(last_driver, $3); }
```

Comment:
Can you rename the setters too? log_threaded_dest_driver_set_flush_lines -> ..._set_batch_lines